### PR TITLE
mr: add page

### PR DESCRIPTION
**Pull request by majecek**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/mr.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> Myrepos manages all your version control repositories at once
+
+- Register repository:
+
+`mr register`
+
+- Update repository in 5 concurent jobs:
+
+`mr -j5 update`
```
Comment (agnivade) [inline pages/common/mr.md]: Wrap 5 in tokens like `-j{{5}}`

File: pages/common/mr.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Prints status of my repos:
```
Comment (agnivade) [inline pages/common/mr.md]: Should it be "Print statuses of all repos" ?

File: pages/common/mr.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`mr status`
+
+- Checkout my repos to latest version:
```
Comment (agnivade) [inline pages/common/mr.md]: Same here, can the word "my" be removed ?

File: pages/common/mr.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# mr
+
+> Myrepos manages all your version control repositories at once.
+
+- Register repository:
```
Comment (sbrl) [inline pages/common/mr.md]: Perhaps `Register a repository` would read better here?

File: pages/common/mr.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+`mr register`
+
+- Update repository in 5 concurent jobs:
```
Comment (sbrl) [inline pages/common/mr.md]: Perhaps `Update repositories`.... would aid clarity here?

File: pages/common/mr.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+`mr -j{{5}} update`
+
+- Print statuses of all repos:
```
Comment (sbrl) [inline pages/common/mr.md]: Perhaps `Print the status of all repositories` would read better here?

File: pages/common/mr.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+`mr status`
+
+- Checkout repos to latest version:
```
Comment (sbrl) [inline pages/common/mr.md]: Perhaps `Checkout all repositories to the latest version` would read better?

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @majecek :D

**Pull request merged**